src/group.ts

```diff
@@ -539,3 +274,19 @@ export class Group {
         return Scalar.hash(this, msg, dst)
     }
 }
+
+type CurveDef = {
+    fp: Field<bigint>
+    Point: ProjConstructor<bigint>
+    h2c: typeof hashToP256
+}
+const curves: Record<GroupID, CurveDef> = {
```

Comment:
are ristretto and decaf groups also supported in noble package? voprf specification also admits these groups.

Reply:
ristretto yeah